gha/py-dependencies/action.yaml

```diff
@@ -7,10 +7,12 @@ description: |
   StackStorm-Exchange/ci/gha/apt-dependencies.
 author: StackStorm
 
-inputs:
+
+s:
```

Comment:
Typo?
```suggestion
inputs:
```

Reply:
Yes, typo. Complete accident. Didn't even know it happened. Resolved in a recent commit.